ipaserver/plugins/baseldap.py

```diff
@@ -550,6 +550,7 @@ class LDAPObject(Object):
     uuid_attribute = ''
     attribute_members = {}
     rdn_is_primary_key = False # Do we need RDN change to do a rename?
+    allow_rename = False
```

Comment:
Because we want to rename objects whose primary key is not RDN. But you're right, perhaps the attribute is just ill-named.